Description: autotools: add zcat
 Add zcat as alternative fot gzcat: in Debian zcat is still in used.
 It is believed that zcat is still widely used, as such it is not
 a Debian specific fix.
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2016-02-20

--- a/configure.ac
+++ b/configure.ac
@@ -203,6 +203,7 @@
 AX_NEED_PROG([UNAME], [uname], [])
 AX_NEED_PROG([UNIQ], [uniq], [])
 AX_NEED_PROG([WC], [wc], [])
+AX_CHECK_PROG([ZCAT], [zcat], [])
 AX_CHECK_PROG([ZCAT], [gzcat], [])
 AX_CHECK_PROG([ZCAT], [gzip], [-dc])
 fi
@@ -303,6 +304,7 @@
 AX_NEED_PROG([TR], [tr], [])
 AX_CHECK_PROG([UNZIP], [unzip], [])
 AX_NEED_PROG([WC], [wc], [])
+AX_CHECK_PROG([ZCAT], [zcat], [])
 AX_CHECK_PROG([ZCAT], [gzcat], [])
 AX_NEED_PROG([ZCAT], [gzip], [-dc])
 fi
